src/entities/session.entity.ts

```diff
@@ -16,10 +17,9 @@ export class Session {
   refreshToken: string;
 
   @Column({
-    type: "text",
-    nullable: true
+    type: "jsonb",
   })
-  device: string;
+  device: deviceInterface;
```

Comment:
don't use interface, use class for that in session.entity file

Reply:
Why is it better to use class?

Reply:
Because use can add decorators )

Reply:
I've added separate decorator for the endpoint , is it ok?